composer.json

```diff
@@ -78,7 +78,8 @@
         "symfony/filesystem": "^4.4",
         "symfony/maker-bundle": "^1.17",
         "symfony/phpunit-bridge": "^5.1.1",
-        "symfony/yaml": "^4.4"
+        "symfony/yaml": "^4.4",
+        "vimeo/psalm": "^3.13"
```

Comment:
I'm just thinking about it.

We're requiring phpstan/phpstan. But the action is also using `phpstan/phpstan-symfony`, etc...
If we can't use phpstan locally without the plugin, maybe we should require them too.

Reply:
I'm adding phpstan and psalm Symfony plugins